scripts/linux/install-kubernetes.sh

```diff
@@ -13,16 +13,32 @@ while true; do
   esac
 done
 
+verbose_output="$1"
+
+echo "verbose_output: $verbose_output"
+
+verbose_flag=
+if [ "$verbose_output" = "on" ]; then
```

Comment:
Sorry, I noticed just now :(

Can we use true/false instead of on/off? I mean the truthy values, not the string.